drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -1047,6 +1092,8 @@ static const struct v4l2_ioctl_ops bcm2835_isp_node_ioctl_ops = {
 	.vidioc_enum_fmt_vid_out	= bcm2835_isp_node_enum_fmt,
 	.vidioc_enum_fmt_meta_cap	= bcm2835_isp_node_enum_fmt,
 
```

Comment:
Does it need this blank line? It groups nicely with the previous lines.

Reply:
Yes will do.